### DISCUSSION TITLE
bash: vercel: command not found

### DISCUSSION THREAD

**Discussion by c21xdx** [vercel/vercel#7593] [Help] [answered]
failed to install CLI, what is wrong with me? thanks
![nodejs](https://user-images.githubusercontent.com/5766149/159304477-72a08566-deb5-4d95-897c-f05d1eab0a85.jpg)

**Comment by c-pederneschi** [comment]
did you resolved the issue ? how you did it ?

**Comment by amyegan** [comment, marked as answer]
This seems to be related to https://github.com/vercel/vercel/discussions/5019. The problem may be caused by the way node/npm were installed which result in unexpected path differences. The solution is to add `vercel` to `PATH` as described in https://github.com/vercel/vercel/discussions/5019#discussioncomment-333260 and https://github.com/vercel/vercel/discussions/5019#discussioncomment-273434
